napari/utils/notifications.py

```diff
@@ -397,9 +397,9 @@ def show_console_notification(notification: Notification):
         ):
             return
 
-        print(notification)
+        print(notification)  # noqa: T201
     except Exception:
-        print(
+        print(  # noqa: T201
```

Comment:
I prefer to change this to [`logging.exception`](https://docs.python.org/3.8/library/logging.html#logging.exception)

Reply:
@brisvag and this?

Reply:
But warnings should show up as popups right?

Reply:
yes. But see code above. There is only settings interaction. So if interaction with settings object fail, you expect GUI to be working?

Reply:
I guess no :P but a `warnings.warn` will be rendered on the terminal as well right?

Reply:
Lets do this this way

Reply:
Or maybe not. The `logging.exception` will also print exception information. There is no equivalent for warnings. If you prefer warning, then we should print stacktrace somehow.

Reply:
ok, logging is fine by me then. We should really get a custom handler going afterwards :)

Reply:
but do we still raise after? I guess so?

Reply:
See comment above raise.